metadeploy/management/commands/extract_labels.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = (
+        "Queries for api_clickthroughagreement records and outputs them in csv format"
+    )
+
+    translatable_objects = {
+        "Product": {
+            "fields": [
+                "title",
+                "short_description",
+                "description",
+                "click_through_agreement",
+                "error_message",
+            ]
+        },
+        "PlanTemplate": {
+            "fields": ["preflight_message", "post_install_message", "error_message"]
+        },
+        # "Plan": {
+        # "fields": [
+        # "title",
+        # "preflight_message_additional",
+        # "post_install_message_additional",
+        # ]
+        # },
+        # "Step": {"fields": ["name", "description"]},
+        "SiteProfile": {
+            "fields": ["name", "company_name", "welcome_text", "copyright_notice"]
+        },
+    }
+
+    field_descriptions = {
+        "Product": {
+            "title": "The name of the product",
+            "short_description": "Short Description of the product",
+            "description": "Description of the Product",
+            "click_through_agreement": "Users must check a box to agree to this legal text before running the installer",
+            "error_message": "Message regarding what to do if an error occurs during installation",
+        },
+        "PlanTemplate": {
+            "preflight_message": "Message displayed during preflight checks",
+            "post_install_message": "Message displayed after installation completes",
+            "error_message": "Message displayed to users if an error occurs",
+        },
+        # "Plan": {
+        # "title": "Title of the plan",
+        # "preflight_message_additional": "An additional preflight message",
+        # "post_install_message_additional": "An additional post install message",
+        # },
+        # "Step": {
+        # "name": "The name of the step",
+        # "description": "Description of the step",
+        # },
+        "SiteProfile": {
+            "name": "Name of the site profile",
+            "company_name": "Name of the company",
+            "welcome_text": "Text shown above products on the home page",
+            "copyright_notice": "Copyright notice shown in the footer of the page",
+        },
+    }
+
+    def handle(self, *args, **options):
+        translatable_labels = {}
+        for obj in self.translatable_objects.keys():
```

Comment:
```suggestion
        for obj in self.translatable_objects:
```

(iterating over a dict actually iterates over its keys implicitly -- if you prefer to be explicit that's fine though)